src/app/components/account-total-balance.tsx

```diff
@@ -1,26 +1,36 @@
 import { memo } from 'react';
 
-import { styled } from 'leather-styles/jsx';
+import { css } from 'leather-styles/css';
+import { type BoxProps, styled } from 'leather-styles/jsx';
 
 import { useTotalBalance } from '@app/common/hooks/balance/use-total-balance';
 
-import { shimmerStyles } from '../../../theme/global/shimmer-styles';
+import { shimmerStyles } from '../../shared/shimmer-styles';
+import { BalanceShimmer } from './balance/balance-shimmer';
 
-interface AccountTotalBalanceProps {
+interface AccountTotalBalanceProps extends BoxProps {
```

Comment:
I don't think you need `extends BoxProps`

Reply:
yes, removed